app/presenters/tree_builder_alert_profile_obj.rb

```diff
@@ -0,0 +1,59 @@
+class TreeBuilderAlertProfileObj < TreeBuilder
+  def initialize(name, type, sandbox, build = true, assign_to: nil, cat: nil, objects: nil)
+    @assign_to = assign_to
+    @cat = cat
+    @objects = objects
+    @cat_tree = true if @assign_to.ends_with?("-tags")
+    super(name, type, sandbox, build)
+  end
+
+  def override(node, object, _pid, _options)
+    identifier = (object.name.presence || object.description)
+    identifier += "-" + object.hostname if object.kind_of?(MiddlewareServer)
+    node[:title] = identifier
+    node[:hideCheckbox] = false
+    node[:select] = @objects.include?(object.id)
+    node
+  end
+
+  def tree_init_options(_tree_name)
+    {
+      :expand => true
+    }
+  end
+
+  def set_locals_for_render
+    locals = super
+    locals.merge!(
+      :id_prefix   => "obj_treebox2",
```

Comment:
..aand you can just remove `id_prefix` altogether, turns out, it's no longer being used at all :).